docs/release-v1-perf-scale-benchmarks.md

```diff
@@ -0,0 +1,79 @@
+### Benchmarks
+
+This document shares some of the performance benchmarks observed as part of the v1 release.
+It will talk about what this means for an end-user's perf and scale story.
+
+#### Background: How to interpret the numbers?
+
+KubeVirt is an extension for Kubernetes that includes a collection of custom resource definitions
+(CRDs) served by kubevirt-apiserver. These CRDs are managed by controllers. Due to the distributed
+nature of the system, understanding performance and scalability data becomes challenging without
+taking specific assumptions into account. This section aims to provide clarity on those assumptions.
+
+1. The data presented in the document is collected by `periodic-kubevirt-e2e-k8s-1.25-sig-performance`.
+1. The test suite includes three tests:
+   1. It creates 100 minimal VMIs, with a small pause of 100 ms between creation of 2 VMIs. The definition
+      of minimal VMIs can be found here https://github.com/kubevirt/kubevirt/blob/20f6caaba4108733a2c3f216e3247202929c1ef9/tests/performance/density.go#L273
+      This is represented in the graphs as <Metric> for VMI, for example `vmiCreationToRunningSecondsP50` for VMI
+   2. It creates 100 minimal VMs, with a small pause of 100 ms between creation of 2 VMIs. The definition
+      of minimal VMs created can be found here https://github.com/kubevirt/kubevirt/blob/20f6caaba4108733a2c3f216e3247202929c1ef9/tests/performance/density.go#L219C1-L219C1
+      This is represented in the graphs as <Metric> for VM, for example `vmiCreationToRunningSecondsP50` for VM
+   3. It creates VMs with instancetype and preference, the definition can be found here https://github.com/kubevirt/kubevirt/blob/20f6caaba4108733a2c3f216e3247202929c1ef9/tests/performance/density.go#L203
```

Comment:
supernit - Use [here](https://github.com/kubevirt/kubevirt/blob/20f6caaba4108733a2c3f216e3247202929c1ef9/tests/performance/density.go#L203) to avoid github rendering the code snippet when previewing the file.

Reply:
fixed